files/configfile/main.jinja

```diff
@@ -0,0 +1,52 @@
+{% set ip_addresses = salt['network.ip_addrs']('include_loopback=True') %}
+{% set smtp_version = salt['cmd.run']('rpm --qf \'%{version}\' -q postfix') %}
+{% set mduri = 'http://169.254.169.254' %}
+{% set mdpath = mduri + '/latest/dynamic/instance-identity/document/' %}
+{% set awsreg = salt['cmd.run']('curl -s ' + mdpath + ' | awk \'/"region"/{print $3}\' | sed -e \'s/"//g\' -e \'s/,//g\'') %}
```

Comment:
ok. you _can_ get this from salt grains, just need to enable metadata grains in the minion config (probably on the cfn-pwm side). can leave it like this for now though.

https://docs.saltstack.com/en/latest/ref/grains/all/salt.grains.metadata.html#module-salt.grains.metadata